LICENSE

```diff
@@ -1584,3 +1584,28 @@ The externally maintained libraries used by Node.js are:
     OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
     WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   """
+
+- fs-extra, located at lib/internal/fs/copy, is licensed as follows:
```

Comment:
The process to update LICENSE is to change and run https://github.com/nodejs/node/blob/master/tools/license-builder.sh
You'll need to check in a copy of the license file from fs-extra to do that